package.nls.zh-cn.json

```diff
@@ -433,7 +433,8 @@
     "DataScience.loadClassFailedWithNoInternet": "加载错误 {0}:{1}。需要连接互联网才能加载第三方控件。",
     "DataScience.useCDNForWidgets": "控件需要从第三方网站下载支持文件。点击[这里](https://aka.ms/PVSCIPyWidgets)获取更多信息。",
     "DataScience.loadThirdPartyWidgetScriptsPostEnabled": "改变 \"jupyter.widgetScriptSources\" 的设置后请重新启动内核。",
-    "DataScience.enableCDNForWidgetsSetting": "控件需要从第三方网站下载支持文件。点击<a href='https://command:jupyter.enableLoadingWidgetScriptsFromThirdPartySource'>这里</a>启用此功能，或点击<a href='https://aka.ms/PVSCIPyWidgets'>这里</a>获取更多信息。（加载错误 {0}:{1}）。",
+    "DataScience.enableCDNForWidgetsSettingHtml": "控件需要从第三方网站下载支持文件。点击<a href='https://command:jupyter.enableLoadingWidgetScriptsFromThirdPartySource'>这里</a>启用此功能，或点击<a href='https://aka.ms/PVSCIPyWidgets'>这里</a>获取更多信息。（加载错误 {0}:{1}）。",
```

Comment:
Do you speak Mandarin too? Or did google translate do this?

Reply:
I'm a man of many talents

Reply:
One of them is `copy paste`, i just copied the previous line. The only different was I removed the HTML tags, thats it.